app/utils/discogs.ts

```diff
@@ -1,5 +1,7 @@
 import { z } from "zod";
```

Comment:
this `utils` folder should go in the root too, same thing with the `globals.css` file (in a `styles` folder like I had mentioned)

Reply:
Oops I thought I did that in my previous commits